Server/object/response.js

```diff
@@ -0,0 +1,8 @@
+class Response {
+    constructor(status, body) {
+      this.status = status;
+      this.body = body;
+    }
+  }
+
+exports.Response = Response;
```

Comment:
Please remove this. You can just pass a normal object instead of creating a class.
```suggestion
```